src/Util/Emailer.php

```diff
@@ -116,7 +116,7 @@ public function send(IEmail $email): bool
 	{
 		Hook::callAll('emailer_send_prepare', $email);
 
-		if (empty($email)) {
+		if (! $email instanceof IEmail) {
```

Comment:
Is this ever going to be false according to the method's type hint?

Reply:
Yes, because of the referenced call in `Hook::callAll()`.